src/app/features/activity-list/activity-list.tsx

```diff
@@ -1,18 +1,21 @@
 import { useAccountTransactionsWithTransfers } from '@app/common/hooks/account/use-account-transactions-with-transfers.hooks';
+import { useGetTransactionsByAddress } from '@app/query/bitcoin/address/address.hooks';
 import { useCurrentAccountFilteredMempoolTransactionsState } from '@app/query/stacks/mempool/mempool.hooks';
-import { TransactionsList } from '@app/features/activity-list/components/transactions-list';
 import { useSubmittedTransactions } from '@app/store/submitted-transactions/submitted-transactions.selectors';
+import { BITCOIN_TEST_ADDRESS } from '@shared/constants';
 
 import { NoAccountActivity } from './components/no-account-activity';
-import { SubmittedTransactions } from './components/submitted-transactions';
+import { SubmittedTransactions } from './components/submitted-transactions/submitted-transactions';
+import { TransactionsList } from './components/transactions/transactions-list';
 
 export const ActivityList = () => {
-  const transactions = useAccountTransactionsWithTransfers();
-  const pendingTransactions = useCurrentAccountFilteredMempoolTransactionsState();
+  const bitcoinTransactions = useGetTransactionsByAddress(BITCOIN_TEST_ADDRESS);
+  const stacksTransactions = useAccountTransactionsWithTransfers();
+  const pendingStacksTransactions = useCurrentAccountFilteredMempoolTransactionsState();
   const submittedTransactions = useSubmittedTransactions();
 
-  const allTransactions = [...pendingTransactions, ...transactions];
-  const hasTransactions = allTransactions.length > 0;
+  const allStacksTransactions = [...pendingStacksTransactions, ...stacksTransactions];
+  const hasTransactions = allStacksTransactions.length > 0 || bitcoinTransactions?.length > 0;
```

Comment:
Maybe this should use a `useMemo`, spreading in quick, but it's still quite a lot to do on every render